tensorflow/lite/interpreter_test.cc

```diff
@@ -245,23 +245,29 @@ TEST(BasicInterpreter, CheckResize) {
     ASSERT_EQ(interpreter.AddTensors(2), kTfLiteOk);
     interpreter.SetInputs({0, 1});
     interpreter.SetOutputs({});
-    TfLiteQuantizationParams quant;
 
+    TfLiteQuantization quantization;
+    auto* quant_param = reinterpret_cast<TfLiteQuantizationParams*>(
```

Comment:
This isn't the intended usage of the quantization params.

TfLiteQuantization has two fields, type and params. 

// Structure specifying the quantization used by the tensor, if-any.
typedef struct {
  // The type of quantization held by params.
  TfLiteQuantizationType type;
  // Holds a reference to one of the quantization param structures specified
  // below.
  void* params;
} TfLiteQuantization;

I need to update the comment, but only AffineQuantization will be used in this structure. The TfLiteQuantizationParams is a separate filed and will still be compatible in the future, since we have backwards compat promises. For that reason I do not think this particular PR is worthwhile, since the old structure is simpler to write for the purposes of this test. Hope that makes sense. Thanks!

Reply:
@suharshs , thanks for review comments, i think you made perfect sense, I am closing this PR as per your suggestion. 

I have one more request, there are some of my important PRs on Quantization under your name, can you please have a look at those as well.

Regards
Amit